README.md

```diff
@@ -25,9 +25,11 @@ gem 'twingly-search'
 require 'twingly/search'
 
 client = Twingly::Search::Client.new
```

Comment:
> I'm thinking we need to promote it if we want our customers to set a custom user agent.

Hmm, do we want our customers to do this? It's nice to see which of our clients/versions they are using.